client/state/support/selectors.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal dependencies
+ */
+import { SUPPORT_SESSION_STATE } from './reducer';
+
+export function isSupportSession( state ) {
+	const { supportSession } = state.support;
```

Comment:
just a meaningless nitpick, but since we're doing destructuring already, is there a reason not to put it into the function arguments?

```js
export function isSupportedSession( { support: { supportSession: state } } ) {
	return state === SUPPORT_SESSION_STATE.ACTIVE || …
}
```

or simplify further…

```js
export const isSupportSession = ( { support: { supportSession: state } } ) =>
	state === SUPPORT_SESSION_STATE.ACTIVE ||
	state === SUPPORT_SESSION_STATE.EXPIRED;
```

Reply:
Thanks, I'm adopting the argument destructuring but generally favor named functions because function names are reflected in the debugger calls stacks. And AFAIK debuggers aren't typically able to associate names with arrow functions to display them in the call stack UI.

In this case, it doesn't really matter because it is a simple expression containing no calls of it's own. Is there a reason arrow functions should be preferred instead?

Reply:
no reason to prefer named functions here as it was my style. I like them for a variety of reasons, mainly that they prevent the confusion of `this` and therefore maintain a simpler model of computation. `function` though is static and available in the entire module scope whereas `const` is only available after being defined.

the point was destructuring though, not the arrow function preference 😄